components/Document/PaperMetadataForm.tsx

```diff
@@ -11,20 +11,108 @@ import updatePaperMetadataAPI from "./api/updatePaperMetadataAPI";
 import colors from "~/config/themes/colors";
 import HubSelectDropdown from "../Hubs/HubSelectDropdown";
 import { ClipLoader } from "react-spinners";
+import { components } from "react-select";
+import useCurrentUser from "~/config/hooks/useCurrentUser";
+import FormSelect from "../Form/FormSelect";
 const { setMessage, showMessage } = MessageActions;
 
+// These options match the OpenAlex types: https://api.openalex.org/works?group_by=primary_location.license:include_unknown
+const LICENSE_OPTIONS = [
```

Comment:
Can we move this list to its own file? Thinking something like `licenseOptions.ts`. I definitely see us using these license options elsewhere such as in the notebook right before publishing content.

Reply:
Moved it!